constants/index.ts

```diff
@@ -0,0 +1,5 @@
+export const supportedChains = [
+  'ecash',
+  'bitcoincash'
+] // WIP integration tests to check if this corresponds to the database
+export const supportedAddressesPattern = /(q|p)[a-z0-9]{41}/
```

Comment:
Lets get more specific with supported addresses. Heres the address format spec
[https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/src/cashaddr.cpp#L16](https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/src/cashaddr.cpp#L16)

relevant part: `const char *CHARSET = "qpzry9x8gf2tvdw0s3jn54khce6mua7l";`

Reply:
An equally valid charset is all uppercase. An address cannot mix upper and lower case though

`ecash:QPZRY9X8GF2TVDW0S3JN54KHCE6MUA7L` is valid
`ecash:QPZRY9X8GF2TVDW0S3jn54khce6mua7l` is not